sopel/config/core_section.py

```diff
@@ -620,18 +856,48 @@ def homedir(self):
 
     :default: ``6667`` normally; ``6697`` if :attr:`use_ssl` is ``True``
 
-    **Required.**
+    .. highlight:: ini
+
+    **Required**::
+
+        port = 6667
+
+    And usually when SSL is enabled::
+
+        port = 6697
+        use_ssl = yes
+
     """
 
     prefix = ValidatedAttribute('prefix', default='\\.')
     """The prefix to add to the beginning of commands.
 
     :default: ``\\.``
 
-    **Required.**
+    .. highlight:: ini
+
+    **Required**::
+
+        prefix = \\.
 
     It is a regular expression (so the default, ``\\.``, means commands start
     with a period), though using capturing groups will create problems.
+
+    With the default value, user will invoke commands like this::
+
+        <nick> .help
```

Comment:
I'm too lazy today to clone the PR and run Sphinx build, but won't the above `.. highlight:: ini` also try to act on this (and the similar one below with `?`)?

Reply:
Aw. It colors it light blue. Not sure I like it this way.

Reply:
Perhaps this section just needs to use explicit `.. code-block::` blocks with the highlighting set for each. :/